CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v4.1.5
+
+ - Added a new output parameter, `wrap_func_args`, which wraps function arguments in parenthesis to speed up parsing in some browsers.
+
```

Comment:
I also deal with the changelog, no worries here. You can delete these changes :)